apps/researcher/src/app/[locale]/objects/settings-menu.tsx

```diff
@@ -26,13 +26,15 @@ export default function SettingsMenu() {
           <button
             onClick={() => viewChange(ListView.Grid)}
             className="p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-neutral-200/50 hover:bg-neutral-300/50 text-neutral-800 transition flex items-center gap-1"
+            aria-label={t('gridAria')}
```

Comment:
We use the prefix `accessibility` for aria labels because the translation keys should be understood by non-programmers, and the term `aria` is not a common term outside of code.